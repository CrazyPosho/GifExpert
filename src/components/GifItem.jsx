import PropTypes from "prop-types";
export const GifItem = ({ title, url }) => {
  return (
    <>
      <div className="card">
        <img src={url} alt={title} className="card-img-top" />
        <p className="card-title">{title}</p>
      </div>
    </>
  );
};

GifItem.propTypes = {
  title: PropTypes.string.isRequired,
  url: PropTypes.string.isRequired,
};
