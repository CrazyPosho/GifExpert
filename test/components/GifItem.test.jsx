const { render } = require("@testing-library/react");
import React from "react";
import { GifItem } from "../../src/components/GifItem";

describe("Pruebas en <GifItem/>", () => {
  const title = "santos";
  const url = "https://elpepe.com";
  test("debe de hacer match con el snapshot", () => {
    const { container } = render(<GifItem title={title} url={url} />);
    expect(container).toMatchSnapshot();
  });
});
