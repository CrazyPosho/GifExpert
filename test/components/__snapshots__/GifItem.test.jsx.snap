// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Pruebas en <GifItem/> debe de hacer match con el snapshot 1`] = `
<div>
  <div
    class="card"
  >
    <img
      alt="santos"
      class="card-img-top"
      src="https://elpepe.com"
    />
    <p
      class="card-title"
    >
      santos
    </p>
  </div>
</div>
`;
